utility/brave_content_utility_client.h

```diff
@@ -15,10 +15,6 @@ class BraveContentUtilityClient : public ChromeContentUtilityClient {
   BraveContentUtilityClient();
```

Comment:
Do we still need to subclass `ChromeContentUtilityClient`? `BraveContentUtilityClient` basically does nothing right now.

Reply:
cc @bridiver

Reply:
I leave it as is because it was added already first in https://github.com/brave/brave-core/commit/9e60026efc5fc475db194f0a9c6180991fcacb45#diff-90a3069fa864dd60b7e276e58817d0ec back when we don't have any real usage yet. If we think it won't be useful in the future, we could consider removing it entirely, which I prefer to do it in a follow-up PR. cc @bridiver